src/Kinematic1D.jl

```diff
@@ -23,6 +21,10 @@ const CMNe = CM.MicrophysicsNonEq
 const CM0 = CM.Microphysics0M
 const CM1 = CM.Microphysics1M
 
+include("Parameters.jl")
+import .Parameters as KP
+const APS = KP.AbstractKinematicParameters
```

Comment:
APS is not used anywhere. Consider removing this line

Reply:
done